browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -356,16 +356,16 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
     {"braveHelpTipsClosingLastTab", IDS_SETTINGS_CLOSING_LAST_TAB_OPTION_LABEL},
     {"braveDisableClickableMuteIndicators",
      IDS_SETTINGS_DISABLE_CLICKABLE_MUTE_INDICATORS},
-  // Leo Assistant Page
-#if BUILDFLAG(ENABLE_AI_CHAT)
```

Comment:
Should probably remove the associated include (`#include "brave/components/ai_chat/common/buildflags/buildflags.h"`)

Reply:
Shure thanks